requirements.txt

```diff
@@ -2,5 +2,5 @@ django>=2.2,<3.1
 django-leaflet>=0.23,<0.27
 channels>=2.1.3,<2.5.0
 Pillow>=6.2.0,<=7.1.0
-openwisp-utils>=0.4.0,<0.5.0
+openwisp-utils~=0.6.0
 geopy>=1.20.0,<1.22.0
```

Comment:
geopy current latest version is 2.0, can you make sure we use that and the code works please?